packages/adapters-cli/src/commands/library-commands.ts

```diff
@@ -10,6 +11,9 @@ import type { Command } from 'commander'
 import { startRpcSnapshot } from '../utils/rpc-interceptor.js'
 import { extractRpcMetrics } from './build-scoreboard-command.js'
 
+const DEFAULT_USER_ADDRESS = '0x47ab2ba28c381011fa1f25417c4c2b2c0d5b4781'
```

Comment:
Any reason to have a default user address? This can be annoying if such address position changes over time.